include/heyoka/func.hpp

```diff
@@ -360,6 +367,24 @@ struct HEYOKA_DLL_PUBLIC_INLINE_CLASS func_inner final : func_inner_base {
             throw not_implemented_error("double eval is not implemented for the function '" + get_name() + "'");
         }
     }
+    long double eval_ldbl(const std::unordered_map<std::string, long double> &m, const std::vector<long double> &pars) const final
+    {
+        if constexpr (func_has_eval_dbl_v<T>) {
```

Comment:
Need to create a new type trait ``func_has_eval_ldbl_v<T>`` here.

Reply:
right, that explains the fact it would not compile when only one was implemented!!! cheers, sorry for not noticing.